test/regression.uts

```diff
@@ -8716,7 +8716,7 @@ assert(rss == ("CON:" if six.PY2 else "foo.exe:"))
 
 random.seed(0x2807)
 rts = RandTermString(4, "scapy")
-assert(sane(raw(rts)) in ["...[scapy", "......scapy"])
+assert(sane(raw(rts)) in ["...Zscapy", "$#..scapy"])
```

Comment:
One will notice that current implementation is broken:
e.g. on python 3, the random string should have a length of 4, + "scapy".
`"......scapy"` is wrong